core/domain/collection_domain.py

```diff
@@ -393,6 +408,17 @@ def from_dict(cls, skill_id, skill_dict):
 
     @staticmethod
     def get_skill_id_from_index(index):
+        """Returns the skill ID from the given index.
+
+        Args:
+            index: str. The index of the collection skill.
```

Comment:
Wrong type, I think. How did you check it?

Reply:
Sorry, it has been corrected to `int` as evident from other methods in the file where index is used.